waku/node/v2/wakunode2.nim

```diff
@@ -206,12 +211,25 @@ proc mountFilter*(node: WakuNode) =
   node.switch.mount(node.wakuFilter)
   node.subscriptions.subscribe(WakuFilterCodec, node.wakuFilter.subscription())
 
-proc mountStore*(node: WakuNode) =
+proc mountStore*(node: WakuNode, path = ""): Result[void, WakuNodeError] =
+  var db: DbConn
+  try:
+    db = db_sqlite.open(path, "", "", "")
+  except:
+    return err(FailedToOpenDatabase)
+  
   info "mounting store"
-  node.wakuStore = WakuStore.init(node.switch, node.rng)
+
+  let store = WakuStore.init(node.switch, node.rng, db)
```

Comment:
What if I'm a node that only wants to request data through the store protocol, but not actually store it?